tests/test_experimentsdaily.py

```diff
@@ -0,0 +1,46 @@
+import json
+import os
+import pytest
+from pyspark.sql.types import StructType
+
+
+# XXX Use the actual numbers.
+EXPECTED_INTEGER_VALUES = {
+    'active_addons_count_mean': 0,
+    'crashes_detected_content_sum': 0,
+    'first_paint_mean': 0,
+    'pings_aggregated_by_this_row': 0,
+    'search_count_all_sum': 0
+}
+
+
+@pytest.fixture
+def make_frame(spark):
+    root = os.path.dirname(__file__)
+
+    schema_path = os.path.join(root, 'resources',
+                               'experiments-summary.schema.json')
+    with open(schema_path) as f:
+        d = json.load(f)
+        schema = StructType.fromJson(d)
+    rows_path = os.path.join(root, 'resources',
+                             'experiments-summary-190-rows.json')
+    frame = spark.read.json(rows_path, schema)
+    return frame
+
+
+def test_rollup(spark):
+    from mozetl.experimentsdaily import rollup
+    from mozetl.clientsdaily.rollup import extract_search_counts
+    frame = make_frame(spark)
+    searches_frame = extract_search_counts(frame)
+    return
```

Comment:
Please use [the @pytest.mark.skip(reason="...") decorator](https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions) to skip this test.